docs/api.md

```diff
@@ -1198,6 +1201,85 @@ Successful requests will produce a "200 OK" response with the random bytes as he
 There are no standard failure modes for this endpoint.
 
 
+## POST /v1/oauth/sessions/revoke
+
+:lock: Authenticated using an OAuth bearer token, with the `account` scope.
+
+Revokes a session token for an account. This allows a relying service to destroy a session remotely, without granting the service access to the original session token. As with `/v1/session/destroy`, the device must perform the login sequence to obtain a new session token after the session is revoked.
+
+The client derives the session identifier by generating a set of Hawk credentials, then discarding the authentication and bundle keys. The relying service should store this identifier, and present it to the auth server when the user requests to sign out remotely.
```

Comment:
IIUC it's not necessary to actually derive the authentication and bundle keys, you just ask the HKDF for the first 32 bytes.  We may need to adjust documentation in other places to make the notion of "token id" more explicit.